examples/meteringlab/c1/haproxy-ingress/10_haproxy-ingress.svc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: haproxy-ingress
+spec:
+  selector:
+    app.kubernetes.io/name: haproxy-ingress
+  type: LoadBalancer
```

Comment:
we don't want to escape to internet just to come back again, so traffic should flow through vpc peering. Make sure it's internal load balancer, exposing this service in VPC not in Internet, Some annotations might be missing. You can find more in eks documentation regarding load balancers.